x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/index.tsx

```diff
@@ -38,6 +39,31 @@ export function ServiceGroupsList() {
     []
   );
 
+  const { start, end } = useMemo(
+    () =>
+      getDateRange({
+        rangeFrom: 'now-24h',
+        rangeTo: 'now',
+      }),
+    []
+  );
+
+  const { data: servicesCountData = { servicesCounts: {} } } = useFetcher(
+    (callApmApi) => {
+      if (start && end) {
+        return callApmApi('GET /internal/apm/service-groups-services-count', {
```

Comment:
We already have `GET /internal/apm/service-groups`. We should stay consistent with that:

```suggestion
        return callApmApi('GET /internal/apm/service-groups/services-count', {
```

Reply:
Btw. didn't we decide to get rid of the service counts for 8.5?

Reply:
> Btw. didn't we decide to get rid of the service counts for 8.5?

we did, but the code was already there by the time i got involved, so it wasn't any extra effort to keep it.